example/react/src/App.js

```diff
@@ -16,7 +16,7 @@ function App() {
   }, [])
 
   async function handleOAuthLogin(provider) {
-    let { error } = await auth.signIn({ provider })
+    let { error } = await auth.signIn({ provider }, { redirectTo: 'http://localhost:3000/welcome'})
```

Comment:
For some reason I cannot get this to come back to `http://localhost:3000/welcome`.

Every time it is coming to `http://localhost:3000/callback` as specified by the `GOTRUE_EXTERNAL_GOOGLE_REDIRECT_URI` parameter in the docker-compose.


To replicate:

- clone this repo
- run `npm run test:infra` (this will start the "backend")
- visit `http://localhost:9999/authorize?provider=google&redirect_to=http://localhost:3000/welcome`

it will return you to `http://localhost:3000/callback` instead of `http://localhost:3000/welcome`

Reply:
Resolved by setting `GOTRUE_EXTERNAL_GOOGLE_REDIRECT_URI: http://localhost:9999/callback`

OAuth should go back the to GoTrue server, not the website